src/resources/Account/index.ts

```diff
@@ -226,12 +301,37 @@ export interface IAccountLiabilityAutoLoan extends IAccountLiabilityLoan {
 
 export interface IAccountLiabilityStudentLoan extends IAccountLiabilityLoan {
   sub_type: 'federal' | 'private' | null;
-  sequence: number | null,
-  disbursed_at: string | null,
-  expected_payoff_date: string | null,
-  payoff_amount: number | null,
-  payoff_amount_term: number | null,
-  principal_balance: number | null,
+  sequence: number | null;
+  disbursed_at: string | null;
+  expected_payoff_date: string | null;
+  payoff_amount: number | null;
+  payoff_amount_term: number | null;
+  principal_balance: number | null;
+}
+
+export interface IAccountLiabilityStudentLoansDisbursement extends IAccountLiabilityLoan {
+  sequence: number;
+  disbursed_at: string | null;
+  expected_payoff_date: string | null;
+  delinquent_status: string | null;
+  delinquent_amount: number | null;
+  delinquent_period: number | null;
+  delinquent_action: string | null;
+  delinquent_start_date: string | null;
+  delinquent_major_start_date: string | null;
+  delinquent_status_updated_at: string | null;
+  delinquent_history: TDelinquencyHistoryItem[];
+  trended: TTrendedDataItem[];
+}
+
+export interface IAccountLiabilityStudentLoans extends IAccountLiabilityLoan {
+  sub_type: 'federal' | 'private' | null;
+  term_length: undefined;
```

Comment:
@ossiggy , @jmdelcarmen, the base interface has this defined as `number | null`, why is it defined as `undefined` here? Same for the other attributes ... This is breaking our build with this release of your API client.